src/js/htmlSanitizer.js

```diff
@@ -28,6 +28,10 @@ const SVG_ATTR_LIST_RX = new RegExp('^(accent-height|accumulate|additive|alphabe
     'width|widths|x|x-height|x1|x2|xlink:actuate|xlink:arcrole|xlink:role|xlink:show|xlink:title|' +
     'xlink:type|xml:base|xml:lang|xml:space|xmlns|xmlns:xlink|y|y1|y2|zoomAndPan)', 'g');
 
+const ATTR_VALUE_BLACK_LIST_RX = {
+  'href': new RegExp('^(javascript:).*', 'g')
```

Comment:
Our convention is that regular expressions use the literal instead of the constructor function.
```js
var regExp = /foo/g;
```